src/main/java/net/consensys/wittgenstein/protocol/GSFSignature.java

```diff
@@ -452,8 +466,8 @@ public String toString() {
 
   @Override
   public Protocol copy() {
-    return new GSFSignature(nodeCount, threshold, pairingTime, timeoutPerLevelMs, periodDurationMs,
-        acceleratedCallsCount, nodesDown, this.network.networkLatency);
+    return new GSFSignature(nb, network, nodeCount, threshold, pairingTime, timeoutPerLevelMs,
```

Comment:
For a copy you want to use a new network (the network object is 'used' when running a scenario.